project/plugins.sbt

```diff
@@ -17,4 +17,4 @@ addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.0")
 addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "0.6.0")
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.24")
 addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.8")
-addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.1")
+addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.6.0-RC4")
```

Comment:
Beware that v1.6 is incompatible with the version that comes bundled with IntelliJ. It's currently not possible to override the scalafmt version that comes installed by default with IntelliJ https://gitter.im/JetBrains/intellij-scala?at=5bcf1a52ae7be94016a22b6c

Reply:
Yes I have project with 1.6 and it's annoying because it invalidates the use of the Intellij plugin.